src/SFA.DAS.EmployerAccounts.Api.UnitTests/Controllers/EmployerAccountsControllerTests/WhenIGetAccounts.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web.Http.Results;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Queries.GetPagedEmployerAccounts;
+using SFA.DAS.EmployerAccounts.TestCommon.Extensions;
+using SFA.DAS.EmployerAccounts.Api.Types;
+
+namespace SFA.DAS.EmployerAccounts.Api.UnitTests.Controllers.EmployerAccountsControllerTests
+{
+    [TestFixture]
+    public class WhenIGetAccounts : EmployerAccountsControllerTests
+    {
+        [Test]
+        public async Task ThenAccountsAreReturnedWithTheirBalanceAndAUriToGetAccountDetails()
```

Comment:
This API doesn't deal with balances does it?

Reply:
Renamed